pkg/proc/debuginfod/debuginfod.go

```diff
@@ -1,17 +1,26 @@
 package debuginfod
 
 import (
+	"os"
 	"os/exec"
 	"strings"
 )
 
-const debuginfodFind = "debuginfod-find"
+const (
+	debuginfodFind       = "debuginfod-find"
+	debuginfodMaxtimeEnv = "DEBUGINFOD_MAXTIME"
+	debuginfodTimeoutEnv = "DEBUGINFOD_TIMEOUT"
+)
 
 func execFind(args ...string) (string, error) {
 	if _, err := exec.LookPath(debuginfodFind); err != nil {
 		return "", err
 	}
 	cmd := exec.Command(debuginfodFind, args...)
+	if os.Getenv(debuginfodMaxtimeEnv) == "" || os.Getenv(debuginfodTimeoutEnv) == "" {
+		cmd.Env = append(os.Environ(), debuginfodMaxtimeEnv+"=1", debuginfodTimeoutEnv+"=1")
```

Comment:
A 1 second timeout?

Reply:
Yes, it's the minimum possible timeout.

Reply:
We  could set instead DEBUGINFOD_PROGRESS. The problem is that this is still going to be invisible with other frontends (vscode-go, goland), I think. We are strictly limited to what our interface can look like by the existence of those other frontends, in this case.
One second is enough to get already downloaded symbols, if users need more they can opt-in by setting the environment variables explicitly (maybe we should add those to the documentation).

Reply:
That's a good thought (re: DEBUGINFOD_PROGRESS). For IDE integrations, is there any prior art for this, or standard within DAP for the debug server to send a message to the client to display? There are certain events mentioned in the standard such as the Progress Start/Update/End events, but that might be tedious to implement, as we currently don't expose any of that information at all, so plumbing it back up might not be worth it.

@hyangah (I know you've moved on, but maybe can ping other folks) any thoughts?

Reply:
That's not a bad idea. I [tried implementing it](https://github.com/aarzilli/delve/tree/progress-debuginfod) but I couldn't get it to show anything. Either it does nothing (bad) or it does something but it's very subtle (also bad). Or maybe I did something wrong, I don't know.

To try it cd into _fixtures/cgostacktrace, `export PATH=.:$PATH` and `export GODEBUG=execerrdot=0` so that the debuginfod-find script gets run instead of the real debuginfod-find and then run vscode in this environment.